content/items/pyVHDLModel.md

```diff
@@ -11,11 +11,20 @@ tags: [
   "dom",
   "language-model",
   "python",
-  "VHDL",
+  "vhdl",
 ]
 talk: 57
 ---
 
-This is a long description...
-<!--more-->
-... about pyVHDLModel.
+# Main Goals
+This Python package provides a unified abstract language model for VHDL. Projects
+reading from source files can derive own classes and implement additional logic
+to create a concrete language model for their tools.
+
+Projects consuming pre-processed VHDL data (parsed, analyzed or elaborated) can
+build higher level features and services on such a model, while supporting
+multiple frontends.
+
+## Use Cases
+* High-level API for [GHDL's](https://ghdl.github.io/ghdl/) libghdl offered as Python package `pyGHDL.dom`.
```

Comment:
Please, use the following internal refs:

```markdown
- [GHDL]({{< ref "/items/ghdl" >}} "GHDL")
- [pyVHDLParser]({{< ref "/items/gpyvhdlparser" >}} "pyVHDLParser")
```

Reply:
why is there a second text?
`GHDL -> link -> GHDL`

Reply:
changed

Reply:
> why is there a second text?
> `GHDL -> link -> GHDL`

It's the "tip" when you hover the mouse.